docs/examples/plot_ghi_transposition.py

```diff
@@ -0,0 +1,82 @@
+"""
+GHI to POA Transposition
+=========================
+
+Example of generating clearsky GHI and POA irradiance.
+"""
+
+# %%
+# This example shows how to use the
+# :py:meth:`pvlib.location.Location.get_clearsky` method to generate clearsky
+# GHI data as well as how to use the
+# :py:meth:`pvlib.irradiance.get_total_irradiance` function to transpose
+# GHI data to Plane of Array (POA) irradiance.
+
+from pvlib import location
+from pvlib import irradiance
+import pandas as pd
+from matplotlib import pyplot as plt
+
+# For this example, we will be using Golden, Colorado
+tz = 'MST'
+lat, lon = 39.755, -105.221
+
+# Create location object to store lat, lon, timezone
+site = location.Location(lat, lon, tz=tz)
+
+
+# Calculate clear-sky GHI and transpose to plane of array
+# Define a function so that we can re-use the sequence of operations with
+# Different locations
+def get_irradiance(site_location, date, tilt, surface_azimuth):
+    # Creates one day's worth of 10 min intervals
+    times = pd.date_range(date, freq='10min', periods=6*24, tz=tz)
```

Comment:
small tiny nit: could you change `tz` to all caps?
```python
# earlier
TZ = 'MST'

# here
times = pd.date_range(date, freq='10min', periods=6*24, tz=TZ)
```
my reasoning is that to me at least `tz` kinda comes out of no where, it looks like it's in the same scope as the function, but really it's not, it's from the module, so that's why making it all caps might be better?

Or as another option, pass it into the function as an argument, or is it possible to retrieve `tz` from the 1st arg, `site_location`, like `site_location.tz`?

Thanks!

Reply:
+1 for `site_location.tz`